tests/test.sh

```diff
@@ -1,15 +1,22 @@
-# Run from top level directory of repository.
+# Run this script from the top level directory of the repository.
 
 # Uncomment if you want to remove the results directory from the previous test run.
 #rm -rf tests/system/all_sets_results_test/
 
+# 1. Run unit tests
+python -m unittest tests/acme_diags/*/test_*.py
```

Comment:
Need to add this line to run the new unit tests.

We should probably restructure the `/test` dir at some point to separate unit tests and the existing integration tests.